### PR TITLE
coba cetak tanda bukti pdf

### PR DESCRIPTION
**Pull request by agnisnafaz**
ErrorException: file_put_contents(D:\YukKP\be_zisapp\storage\framework\views/e875091c62b37c683139b35181f37f2a2d627130.php): failed to open stream: No such file or directory in file 

errornya itu mas

### CODE REVIEW

File: app/Http/Controllers/transaksi/DonasiController.php [hunk 1]
```diff
@@ -206,21 +203,14 @@ public function cetak_pdf(Request $request)
...
-                        ON muzakis.id_muzaki = donasis.id_muzaki"
-        );
+        //menampilkan hasil detail donasi
+        $donasi1 = DB::select("SELECT * FROM donasis LEFT JOIN muzakis ON donasis.id_muzaki =  muzakis.id_muzaki
+        WHERE detail_donasis.id_donasi = '" . $id . "");
 
-        //perintah cetak pdf
-        $pdf = PDF::loadview('tandaterima', ['donasi' => $donasi])->setPaper('A4', 'potrait');
+        //perintah cetak tanda terima pdf
+        $pdf = PDF::loadview('tandaterima', ['donasi1' => $donasi1])->setPaper('A4', 'potrait');
```
Comment (triandamai) [inline app/Http/Controllers/transaksi/DonasiController.php:213]: file `tandaterima` udah ada di views ?

**Review by triandamai — CHANGES_REQUESTED** [review]
coba cek

File: app/Http/Controllers/transaksi/DonasiController.php (hunk 1 above)
Comment (agnisnafaz) [reply app/Http/Controllers/transaksi/DonasiController.php:213]: sudah ada mas,

**Review by agnisnafaz — COMMENTED** [review]
ini viewnya not found mas

**Review by triandamai — APPROVED** [review]

**Pull request merged**